test/e2e/lib/helpers/sdk.ts

```diff
@@ -11,6 +12,18 @@ export async function flushEvents() {
   await waitForServersIdle()
   const servers = await getTestServers()
 
+  // Flush bridge events
+  if (bridgeEvents) {
+    const events = (await browserExecute(() => {
+      const eventBridge = (window as any).DatadogEventBridge
+      if (!eventBridge) {
+        return []
+      }
+      return eventBridge.events as RumEvent[]
+    })) as Array<{ eventType: string; event: RumEvent }>
+    events.forEach(({ eventType, event }) => bridgeEvents.push(eventType === 'log' ? 'logs' : 'rum', event))
```

Comment:
do we need to differentiate monitoring logs from logger logs?

Reply:
For now the bridge does not support log or monitoring so I'll remove the check